hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,136 @@
+defmodule Hangman.Game do
```

Comment:
This whole module is really, really nice. Well laid out, and nicely split into meaningful functions.

My only comment would be to think about times you ise the word `handle` in a function name. All functions handle their arguments, so it doesn't really mean anything.

### Bonus: +5 for making me happy.